scripts/bundle-build/standard-gradle-build/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+outputDir=$(basename $PWD)-artifacts
+mkdir $outputDir
+./gradlew assemble --no-daemon --refresh-dependencies -Dbuild.snapshot=false -DskipTests=true -Dopensearch.version=$1
```

Comment:
What happens if gradlew build fails? (This scenario isn't isolated only to the default build script)

Reply:
If a particular component fails to build, the entire bundle build should exit.

Reply:
This will abort the shell script because `set -e` and the build will fail, https://intoli.com/blog/exit-on-errors-in-bash-scripts/

Reply:
Forgive my ignorance, but where is the `set -e` invoked that will impact this script execution?  I see it was set in opensearch.sh, does that cascade into the other script executions as well?

Reply:
It has been added as part of #150 :)